platforms/shared/charts/vault-k8s-mgmt/templates/job.yaml

```diff
@@ -30,7 +30,7 @@ spec:
       restartPolicy: OnFailure
       imagePullSecrets:
         - name: {{ $.Values.vault.imagesecretname }}
-      serviceAccountName: {{ $.Values.vault.reviewer_service }}
+      serviceAccountName: {{ $.Values.serviceAccount.name }}
```

Comment:
As vault-auth is created by the same chart, 'vault-auth' can be hardcoded here. Or update the templates/serviceAccount.yaml to use the Helm values to name serviceaccount

Reply:
Done